### ISSUE TITLE
CSCDigiToRawModule deferences invalid edm::Handle

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#30916]
The UBSAN build uncovered the following

```
/data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/3d0af0c821cc86dc2e95a3561ce9bea6/opt/cmssw/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_UBSAN_X_2020-07-24-2300/src/EventFilter/CSCRawToDigi/plugins/CSCDigiToRawModule.cc:172:28: runtime error: reference binding to null pointer of type 'const struct GEMPadDigiClusterCollection'
```

The problem is here with the variable `padDigiClusters` which is an `edm::Handle`
https://github.com/cms-sw/cmssw/blob/master/EventFilter/CSCRawToDigi/plugins/CSCDigiToRawModule.cc#L168-L179

The Handle is only set if `useGEMs_` is true. If it is false, than an invalid `edm::Handle` is deferences and a nullptr is returned via the reference. [See bug report #30915 ]. Once the `edm::Handle` bug is fixed, this code will fail.

**Comment by Dr15Jones** [comment]
assign reconstruction

**Comment by Dr15Jones** [comment]
Fixed by #30925

**Comment by perrotta** [comment]
+1
(see https://github.com/cms-sw/cmssw/issues/30916#issuecomment-664440187, thank you @Dr15Jones )
